crates/cxx-qt-lib/src/serde_impl.rs

```diff
@@ -0,0 +1,190 @@
+// SPDX-FileCopyrightText: 2025 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
Hm, the implementations in this file probably work, however, as mentioned in #943 , the goal with our serde support is to keep maintenance effort low.

We should try to replace these implementations with `from` and `into` implementations wherever possible.